index.html

```diff
@@ -44,9 +44,49 @@
         </div>
       </div>
     </header>
+  
     
 
     <!-- Block banner -->
-    <main class="banner"></main>
+  <main class="banner">
+      <div class="main-container">
+      
+  
+      <div class="column-lg-9">
+        <h1 class="main__title">Fullstack-разработка<br> на JavaScript</h1>
+      
+        <div class="main__text">Пройдя курс Fullstack-разработки на JavaScript ты научишься<br>использовать профессиональные инструменты front-end и back-end<br>разработки React и Node.js, СУБД MySQL и MongoDB.</div>
+    
+      <div>
+        <ul class="main__list">
+          <li class="main__list-platform"><b>Площадка:</b><br>ННГУ</li>
```

Comment:
don't use `br` for this purpose 
https://doka.guide/html/br/

Reply:
in general:
it's not a good idea to use `br` [at all](https://developer.mozilla.org/ru/docs/Web/HTML/Element/br#%D0%BF%D1%80%D0%BE%D0%B1%D0%BB%D0%B5%D0%BC%D1%8B_%D0%B4%D0%BE%D1%81%D1%82%D1%83%D0%BF%D0%BD%D0%BE%D1%81%D1%82%D0%B8)